prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -111,6 +111,7 @@ class FHIRConverter(
         companion object {
 
             private val clientIdHeader = "client_id"
+            private val payloadNameHeader = "payloadname"
```

Comment:
This doesn't need to be done now, but since in the future these header names will be used across multiple services they should be defined in the shared project. It makes it less prone to errors if we have to change them for some reason in the future.

Maybe you could throw a `TODO` on there with your ticket for submissions service updates.